aspnetcore/mvc/views/view-components/sample/ViewCompFinal/Views/Todo/Index.cshtml

```diff
@@ -33,5 +34,7 @@
     }
 </table>
 <div>
-    @await Component.InvokeAsync("PriorityList", new { maxPriority = 2, isDone = false })
+    @*@await Component.InvokeAsync("PriorityList", new { maxPriority = 2, isDone = false })*@
+    <vc:priority-list max-priority="2" is-done="false">
+    </vc:priority-list>
```

Comment:
This looks good. Is the old code commented out intentionally? Do we want that there still?